service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -281,13 +373,19 @@ private HearingBooking buildHearingBooking(YesNo useAllocatedJudge) {
             .build();
     }
 
-    private List<Element<HearingBooking>> createHearingBookings() {
-        return List.of(
+    private List<Element<HearingBooking>> createHearingBookings(boolean addHearing) {
```

Comment:
I don't like the approach of this unit test, passing boolean into the method, seems invisible what this method is doing. I would much more prefer to have tests adding extra one hearing if required.

Reply:
changed as suggested.